lib/npm-auditer.js

```diff
@@ -40,18 +40,32 @@ function printReport(parsedOutput, levels, reportType) {
       printReportObj("NPM audit report JSON:", parsedOutput);
       break;
     case "important": {
-      const vulnerabilities =
-        parsedOutput.advisories || parsedOutput.vulnerabilities;
-      const relevantAdvisories = Object.keys(vulnerabilities).reduce(
-        (acc, advisory) =>
-          levels[vulnerabilities[advisory].severity]
-            ? {
-                [advisory]: vulnerabilities[advisory],
-                ...acc,
-              }
-            : acc,
-        {}
-      );
+      let relevantAdvisories;
+
+      if (parsedOutput.auditReportVersion === 2) {
+        const { vulnerabilities } = parsedOutput;
+        relevantAdvisories = Object.keys(vulnerabilities).reduce(
+          (acc, advisory) =>
+            levels[vulnerabilities[advisory].severity]
+              ? {
+                  [advisory]: vulnerabilities[advisory],
+                  ...acc,
+                }
+              : acc,
+          {}
+        );
+      } else {
+        relevantAdvisories = Object.keys(parsedOutput.advisories).reduce(
+          (acc, advisory) =>
+            levels[parsedOutput.advisories[advisory].severity]
+              ? {
+                  [advisory]: parsedOutput.advisories[advisory],
+                  ...acc,
+                }
+              : acc,
+          {}
+        );
+      }
```

Comment:
```ts
const advisories = parsedOutput.auditReportVersion === 2 ? parsedOutput.vulnerabilities : parsedOutput.advisories;
const relevantAdvisories = Object.keys(advisories).reduce(
           (acc, advisory) =>
             levels[advisories[advisory].severity]
               ? {
                   [advisory]: parsedOutput.advisories[advisory],
                   ...acc,
                 }
               : acc,
           {}
         );
```

Reply:
The PR looks great, maybe this can be simplified? Very very excited to get this in 👍🏻

Reply:
Done